applications/CompressiblePotentialFlowApplication/custom_elements/adjoint_base_potential_flow_element.h

```diff
@@ -90,12 +90,6 @@ class AdjointPotentialFlowElement : public Element
     ///@name Operations
     ///@{
 
-    Element::Pointer Create(IndexType NewId, NodesArrayType const& ThisNodes, PropertiesType::Pointer pProperties) const override;
```

Comment:
Here I remove the Create() and Clone() functions of the base element since it is not meant to be used by itself. Is this allowed?

Reply:
I don't know, let me ping @philbucher and @msandre

Reply:
If the base element can't be standalone used I think it makes sense to remove them.